author/views.py

```diff
@@ -1,3 +1,23 @@
-from django.shortcuts import render
+from django.views import generic
+from .models import Author
+from django.core.paginator import Paginator
 
-# Create your views here.
+class BloggerDetailView(generic.DetailView):
```

Comment:
try to import DetailView directly like this 

from django.views.generic import DetailView

Because from django.views import generic will load all of the content into the current file and we don't need this